### ISSUE TITLE
[promtool] Export metrics in openmetrics format

### ISSUE THREAD

**Issue by thoro** [prometheus/prometheus#12022]
### Proposal

It sometimes happens that metrics are ingested with wrong labels, since there's no facility to update the labels on the file inside the tsdb the next best thing would be to export them and reimport.

The reimport is already solved with the tsdb create-block-from openmetrics command.
The export also works via the query range command.

The Problem is that it's not possible to import the data we got from the export with the backfill command. This would allow to modify the data in a text editor or external program and then simply reimport it into prometheus.

For this to work the query range command would need openmetrics as an additional output format (to json and promql).

**Comment by dgl** [comment]
I think query output in OpenMetrics form is probably generally useful. However for your particular use case you actually want rewriting, rather than export+import, as that way either needs the whole TSDB to be dumped (and that doesn't really scale) or you end up with potentially duplicate data when you import the updated labels.

One option that works now is to use thanos's tooling, for offline block modification, see https://thanos.io/tip/operating/modify-objstore-data.md/ (particularly the relabelling part and "Rewrite Prometheus TSDB blocks" at the end).

**Comment by Thoro** [comment]
For me it would be perfect, I just screwed up 10 minutes of my series when switching exporters and now all of my grafana dashboards with group_left / group-right are screwed since I got suddenly more than one series.

The quick fix was to export the data, and delete it from prometheus. I just need to figure out how to get those 10 min back. It’s only about 100k lines, but the create-blocks-from openmetrics complains about an INVALID token (printing the offending line would also be helpful)
